docs/setup.md

```diff
@@ -16,7 +16,15 @@ Load `pg_stat_monitor` at the start time by adding it to the `shared_preload_lib
     >
     >`pg_stat_monitor` **must** follow `pg_stat_statements`. For example, `ALTER SYSTEM SET shared_preload_libraries = 'foo, pg_stat_statements, pg_stat_monitor'`.
 
-2. Start or restart the `postgresql` instance to apply the changes.
+    !!! warning
+   
+       It's make sence to disable application name tracking as it's expensive feature and may cause perfomance degradation proportional to connections number.
+       ```sql
+       ALTER SYSTEM SET pg_stat_monitor.pgsm_track_application_names = 'no';
+       ```
```

Comment:
@nastena1606 I need your help here, how to make this text part of the warning frame?